src/utilities/rt.tsx

```diff
@@ -124,11 +126,7 @@ export function caseInsensitiveFilterMethod(filter: Filter, rows: any) {
     const id = filter.pivotId || filter.id;
     const value: string = filter.value;
     const lowerCaseInput = value.toLowerCase();
-    return rows[id] !== undefined
-        ? String(rows[id])
-              .toLowerCase()
-              .startsWith(lowerCaseInput)
-        : true;
+    return rows[id] !== undefined ? String(rows[id]).toLowerCase().startsWith(lowerCaseInput) : true;
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)